Source/SimpleActions/Public/Structs/SimpleActionList.h

```diff
@@ -20,8 +20,13 @@ struct SIMPLEACTIONS_API FSimpleActionList
 	
 public:
 
-	UPROPERTY(EditAnywhere, BlueprintReadOnly, Instanced, meta = (ShowOnlyInnerProperties))
```

Comment:
IIRC this was necessary to prevent the actions' properties from being nested in groups.

Reply:
I'll have a butchers, this might not need removal anymore. The specifier CollapseCategories removal might only be needed for USimpleAction. Either this or that makes it a Categoryless mess.